lib/stellar_spectrum/services/get_current_sequence_number.rb

```diff
@@ -0,0 +1,13 @@
+module StellarSpectrum
+  class GetCurrentSequenceNumber
+
+    def self.execute(stellar_client:, channel_account:)
+      account_info = stellar_client.account_info(channel_account)
+      account_info.sequence.to_i
+    rescue Faraday::ClientError => e
+      Log.write("#{Client::LOG_TAG}: Retrying GetCurrentSequenceNumber - #{e.inspect}")
```

Comment:
So that we don't have to put the tag every time, the `LOG_TAG` code should be in the `Log` class so that this will just be:

```suggestion
      Log.write("Retrying GetCurrentSequenceNumber - #{e.inspect}")
```